test/dialect/mysql/test_types.py

```diff
@@ -478,6 +479,34 @@ def test_float_type_compile(self, type_, sql_text):
         self.assert_compile(type_, sql_text)
 
 
+class TypeCompileUUIDMariaDBTest(fixtures.TestBase, AssertsCompiledSQL):
+    __only_on__ = "mariadb"
+
+    @testing.only_if("mariadb>=10.7")
```

Comment:
**Federico Caselli** (CaselIT) wrote:

maybe it can be set on `__requires__`? I see that `__only_on__` accepts a single dialect :(

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/5120

Reply:
Hi, as I understand current updates in code are only in gerrit and I'm not sure if I have permissions to upload code to gerrit.
Please advise if I can help with work on gerrit.

Reply:
This patch is pretty much done, it's not been moving lately as it's targeted at 2.1 only and I've had lots of other things I've had to work on.     What were the changes you were considering?

Reply:
**Michael Bayer** (zzzeek) wrote:

`__only_on__` accepts multiple dialects, look at all the other tests here.  the awkwardness is that I have some compilation-only tests here also that dont need a backend.  but it doesnt matter we can move the only_on upwards

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/5120

Reply:
the comment was for Mike, since he did a pass on this change on gerrit and the bot synchronizes the messages with the PR if it can

You can ignore this. Thanks for the effort so far!

Reply:
**Federico Caselli** (CaselIT) wrote:

oh, it's documented as supporting only a single one! well I guess it was changed at some point and the docs was not updated

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/5120

Reply:
> This patch is pretty much done, it's not been moving lately as it's targeted at 2.1 only and I've had lots of other things I've had to work on. What were the changes you were considering?

Here (on github) i can see only my initial changes and commits.
And also get updated from code review in Gerrit.

If no additional changes needed - it's great! and my question is not relevant.